_includes/php/users.md

```diff
@@ -91,6 +91,7 @@ It would be bothersome if the user had to log in every time they open your app.
 By default, whenever you use any signup or login methods, the user will be saved in PHP Session storage (The `$_SESSION` superglobal.)
 
 ```php
+//Here we need to call session_start() if we want to rerieve the current user
```

Comment:
Can you please move the line out of the code block and add it after the sentence above?

> By default, whenever you use any signup or login methods, the user will be saved in PHP Session storage, the `$_SESSION` superglobal. When re-initializing the Parse PHP SDK, you may need to instruct PHP to load the stored session data into the `$_SESSION` superglobal using `session_start()` before initializing the Parse PHP SDK to make continue with the user session.